packages/jaeger-ui/src/constants/default-config.tsx

```diff
@@ -87,6 +87,22 @@ export default deepFreeze(
         trackErrors: true,
         customWebAnalytics: null,
       },
+      monitor: {
+        menuEnabled: true,
+        emptyState: {
+          mainTitle: 'Get started with Service Monitoring',
+          subTitle:
+            'A high-level monitoring dashboard that helps you cut down the time to identify and resolve anomalies and issues.',
+          description:
+            'Service Performance Monitoring works in conjunction with a Prometheus compatible time series database. The metrics are aggregated from your tracing data through a one-time configuration. ',
+          button: {
+            text: 'Read the Documentation',
+            onClick: () => window.open('https://www.jaegertracing.io/docs/latest/frontend-ui/'),
```

Comment:
changing. BTW we get 404 now
![image](https://user-images.githubusercontent.com/11076023/160274001-c22dcc64-b1fd-4058-8f4f-e45f47510855.png)

Reply:
> BTW we get 404 now

That's fine, because that page hasn't been released yet, but should be released within 2 weeks, which is when Jaeger UI will also be released.